nullable/sqltypes.go

```diff
@@ -0,0 +1,17 @@
+package nullable
+
+import (
+	"time"
+)
+
+// StdTime provides default SQL TIME format
+const StdTime = "15:04:05"
```

Comment:
Why should this be exported, and why isn't it called something like `DefaultSQLTime` if that's what it is?

Reply:
That's another one I left as it was, I think initially since that was a sql specific package, this was done in case the user was dealing with a mysql time, rather than datetime, but since it has become irrelevant, I will get rid of it